extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -85,6 +91,18 @@ class AmpStickyAd extends AMP.BaseElement {
   }
 
   /**
+   * The function that add border-bottom to the body
+   * to compensate the bottom space that was taken by sticky ad.
+   * So that no content would be blocked by sticky ad unit.
+   * @param {number} borderBottom
+   * @private
+   */
+  addBorderBottom(borderBottom) {
+    this.win_.document.body.style.borderBottom =
```

Comment:
This should go to `Viewport` API - it's the most appropriate place for all things viewport/body/documentElement. E.g. let's add `Viewport.setBorderBottom` method there.